src/popcorn-list-movies/test/PopcornListMovies.test.js

```diff
@@ -44,11 +48,23 @@ describe("PopcornListMovies", () => {
 
   it("should return nothing when there are no movies in the homepage", async () => {
     const element = await scopedFixture(
-      html `<popcorn-list-movies></popcorn-list-movies>`
+      html`<popcorn-list-movies></popcorn-list-movies>`
     );
 
-    const popcornListMovies = element.shadowRoot.querySelector('[data-testid="popcorn-list-movies"]');
+    const popcornListMovies = element.shadowRoot.querySelector(
+      '[data-testid="popcorn-list-movies"]'
+    );
 
     expect(popcornListMovies).to.be.null;
   });
-});
\ No newline at end of file
+
+  it("should return a poster from assets folder when the image is N/A", async () => {
+    const element = await scopedFixture(
+      html`<popcorn-list-movies .movies=${movies}></popcorn-list-movies>`
+    );
+
+    const movieImage = element.shadowRoot.querySelectorAll('[data-testid="movieImage"]');

```

Comment:
Because this is a collection of items, maybe a better name will be "movieImages".